### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by ashawkey** [liuxinhai/Point2Sequence#1]
Thanks for sharing the codes of this brilliant work!  
I have tried to run the code following the default settings but got slightly lower accuracy (~92.0%) in classification of ModelNet40 using the hdf5 data without normals, so I was wondering whether you could provide a pretrained model to reproduce the results listed in your paper?   
Thanks a lot!

**Comment by liuxinhai** [comment]
> Thanks for sharing the codes of this brilliant work!
> I have tried to run the code following the default settings but got slightly lower accuracy (~92.0%) in classification of ModelNet40 using the hdf5 data without normals, so I was wondering whether you could provide a pretrained model to reproduce the results listed in your paper?
> Thanks a lot!

**Comment by liuxinhai** [comment]
Pretrained Models under ModelNet40: https://drive.google.com/drive/folders/1OY1MOhmL56yUoVfXmLp1zZqIneN46dPd?usp=sharing

**Comment by ashawkey** [comment]
Thanks for reply!  
But I find there is no code to evaluate using a pretrained model in this branch...
Would you mind update to share that code?

**Comment by liuxinhai** [comment]
@ashawkey you can get the evaluate code by modify the train code or you can follow the evaluation as in PointNet++ https://github.com/charlesq34/pointnet2/blob/master/evaluate.py.

**Comment by zqzqz** [comment]
Hi @ashawkey. I am trying to use the pre-trained model you provided  https://drive.google.com/drive/folders/1OY1MOhmL56yUoVfXmLp1zZqIneN46dPd?usp=sharing and I leverage the evaluation script from PointNet++ https://github.com/charlesq34/pointnet2/blob/master/evaluate.py. I used the default settings (no normals). However, the following errors occur when loading model checkpoints:

```
 W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key layer1/seq_attention/encoder/rnn/lstm_cell/bias not found in checkpoint
 W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key layer1/seq_attention/encoder/rnn/lstm_cell/kernel not found in checkpoint
```

Indeed, the checkpoint has no such variable names, only with:

```
layer1/encoder/rnn/lstm_cell/bias
layer1/encoder/rnn/lstm_cell/kernel
```

Could you provide more guidelines for loading the pre-trained models? Thank you!

tensorflow-gpu: 1.4.0
cuda: 8.0
